pkg/apis/workspace/v1alpha1/workspace_types.go

```diff
@@ -72,7 +72,7 @@ const (
 	WorkspaceComponentsReady     WorkspaceConditionType = "ComponentsReady"
 	WorkspaceRoutingReady        WorkspaceConditionType = "RoutingReady"
 	WorkspaceServiceAccountReady WorkspaceConditionType = "ServiceAccountReady"
-	WorkspaceDeploymentReady     WorkspaceConditionType = "DeploymentReady"
+	WorkspaceReady               WorkspaceConditionType = "Ready"
```

Comment:
After the latest changes, it makes sense to have DeploymentReady and WorkspaceReady separately, since we have endpoints health check after deployment is marked as ready. WDYT?

Reply:
:+1: Agree here, though we could use `WorkspaceReady` now and add `DeploymentReady` later, if need be. I've been generally operating under the idea that conditions represent different elements of the workspace, but not an overall summary of workspace state. E.g. `WorkspaceRoutingReady` might be false even though `WorkspaceDeploymentReady` is true. I'd prefer to limit overlap between components for this reason -- `WorkspaceReady` ends up being 'all other conditions are set'.

Reply:
+1 to have workspaceReady distinguished from DeploymentReady. I wondered exactly the same when changing it for the talk. Could this be done in a distinct PR ? It would also impact issue https://github.com/devfile/kubernetes-api/issues/44#issuecomment-633530139 which I'm currently working on.

Reply:
> Could this be done in a distinct PR ?

Sure!